email-templates/weekly/html.ejs

```diff
@@ -0,0 +1,579 @@
+                        <tr>
+
+                          <td class="most-points-inner">
+                            <table cellpadding="0" cellspacing="0">
+                              <tbody>
+                                <tr>
+                                  <td class="most-points-user" valign="bottom">
+                                    <img src="<%= thumbnail %>"/>
+                                    <div><%= mostReceived.user.canvas_full_name %></div>
+                                  </td> 
+                                </tr>
+                                <tr>
+                                  <td class="most-points-column-outer" style="height:<%= mostPointsHeight %>px;" valign="bottom">
+                                    <div class="most-points-column-inner" style="height:<%= mostPointsHeight %>px;"></div>
+                                  </td>
+                                </tr>
+                                <tr>
+                                  <td class="most-points-value">
+                                    <%= mostPoints %> points
+                                  </td>
+                                </tr>
+                              </tbody>
+                            </table>
+                          </td>
+
+                          <td class="most-points-inner">
+                            <table cellpadding="0" cellspacing="0">
+                              <tbody>
+                                <tr>
+                                  <td class="most-points-user" valign="bottom">
+                                    <img src="<%= baseCollabosphereUrl + resolveAsset('/assets/img/icon_class.png') %>"/>
+                                    <div>Class average</div>
+                                  </td> 
+                                </tr>
+                                <tr>
+                                  <td class="most-points-column-outer" style="height:<%= averagePointsHeight %>px;" valign="bottom">
+                                    <div class="most-points-column-inner" style="height:<%= averagePointsHeight %>px;"></div>
+                                  </td>
+                                </tr>
+                                <tr>
+                                  <td class="most-points-value">
+                                    <%= averagePoints %> points
+                                  </td>
+                                </tr>
+                              </tbody>
+                            </table>
+                          </td>
+
+                        </tr>
+                      </tbody>
+                    </table>
+                  <%
+                  }
+                  -%>
+
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+          </td>
+        </tr>
+
+        <tr>
+          <td class="centered">
+          <% 
+          if (course.engagementindex_url) { 
```

Comment:
@johncrossman Well caught; at present it is initialized nowhere! That's a back-end requirement that didn't surface until I started the front end - thanks for reminding me I need to JIRA it.